geometry_msgs/Point target_pose
---
joint_angles joint_vals